SA/project_documentation/content/time_tracking.typ

```diff
@@ -16,4 +16,8 @@ approximately the available hours. A detailed graphic is shown in
 #figure(
   image("../static/time_spent.png"),
   caption: "Time spent per week for each member."
-) <time-tracking-table>
\ No newline at end of file
+) <time-tracking-table>
+
+It is to be noted that some work went into the project before the semester began,
+e.g., through a meeting with the project advisor. @time-tracking-table depicts
```

Comment:
```suggestion
i.e., the definition of the project's task description. @time-tracking-table depicts
```
Work that was done before the official project start needs to be clearly defined.